src/backend/d3d12/FramebufferD3D12.h

```diff
@@ -41,6 +43,9 @@ namespace d3d12 {
             Device* device;
             DescriptorHeapHandle rtvHeap;
```

Comment:
`DescriptorHeapHandle* rtvHeap = nullptr;` or `DescriptorHeapHandle rtvHeap = {}`?

Actually I'm confused about DescriptorHeapHandle. Where is this `device` coming from??
https://github.com/google/nxt-standalone/blob/master/src/backend/d3d12/DescriptorHeapAllocator.cpp#L27
and also what is the default constructor useful for?